README.md

```diff
@@ -1,75 +1,169 @@
+HOSTNAME="localhost"
+PORT=5432  # default PostgreSQL port
+DATABASE="ert_store"
+USERNAME="my-usrnam"
+PASSWORD="my-passwd"
+
+# The connection string is then:
+export ERT_STORAGE_DATABASE_URL="postgresql://${USERNAME}:${PASSWORD}@${HOSTNAME}:${PORT}/${DATABASE}"
+
+# or
+export ERT_STORAGE_DATABASE_URL="postgresql://my-usrnam:my-passwd@localhost:5432/ert_store"
 ```
 
-Then, using Alembic, update the database schema.
+Then install a compatible SQLAlchemy PostgreSQL driver, such as
+[`psycopg2`](https://pypi.org/project/psycopg2/). We provide the extras
+requirement `postgres` which installs the necessary requirements.
+
 ``` sh
-alembic upgrade head
+pip install ert-storage[postgres]  # from PyPI or...
+pip install -e .[postgres]         # edit mode in current working directory
 ```
 
-And you should be done setting up!
+The `psycopg2` package requires the postgres client development headers to be
+installed on your machine. The
+[`psycopg2-binary`](https://pypi.org/project/psycopg2-binary/) is a precompiled
+version of this package that might be compatible with your OS and PostgreSQL
+version.
+
+### Database setup: Container
+You can use either Docker or `podman` to easily start a postgres instance as a
+container. On modern Linux distributions it is easier to use the `podman`
+project.
 
-# Using
+[See the docker.io page for postgres for more information](https://hub.docker.com/_/postgres/)
 
-Start ERT Storage using Uvicorn. `--reload` means that Uvicorn will detect
-changes in any files that you edit and automatically reload the application.
+#### Podman
 
 ``` sh
-uvicorn ert_storage.app:app --reload
+# Start the container in the background, with password 'password' and forwarding the port 5432 to the host.
+podman run --name ert-storage-pg -e POSTGRES_PASSWORD=password -p 5432:5432 -d docker.io/postgres
+
+# Useful commands
+podman container stop ert-storage-pg
+podman container start ert-storage-pg
+podman container kill ert-storage-pg  # SIGTERM
+podman container rm ert-storage-pg    # Remove the container entirely
+
+# Use with ERT Storage
+export ERT_STORAGE_DATABASE_URL="postgresql://postgres:password@localhost:5432/postgres"
 ```
 
-Navigate to https://localhost:8000/docs to see the Swagger docs
+### Database setup: Linux service
 
-To create migrations, use:
+First, install the dependencies:
 
 ``` sh
-alembic revision --autogenerate --message="[Describe your changes]"
-# Inspect and maybe edit the generated file
-alembic upgrade head
+# RHEL / CentOS / Fedora
+sudo yum install postgresql-server postgresql-devel
+
+# Ubuntu
+sudo apt install postgresql postgresql-client
 ```
 
-Sometimes you'll need to drop and recreate the database:
+Then, initialize the database and configure a user with a password.
 
 ``` sh
-dropdb -U pg_ert db_ert
-createdb -U pg_ert db_ert
-alembic upgrade head
+# Initialise stuff in /var
+sudo su -i -l postgres -- initdb
+
+# Create superuser
+sudo su -i -l postgres -- createuser -s ert
```

Comment:
Instead of `ert` as the user (ert is well known abbreviation), can we use `ert_user` or `ert_db_user`?